core/dbt/config/project.py

```diff
@@ -645,13 +645,25 @@ def from_project_root(
     def hashed_name(self):
         return hashlib.md5(self.project_name.encode('utf-8')).hexdigest()
 
-    def get_selector(self, name: str) -> SelectionSpec:
+    def get_selector(self, name: str) -> Union[SelectionSpec, bool]:
         if name not in self.selectors:
             raise RuntimeException(
                 f'Could not find selector named {name}, expected one of '
                 f'{list(self.selectors)}'
             )
-        return self.selectors[name]
+        return self.selectors[name]["definition"]
+
+    def get_default_selector(self) -> Union[SelectionSpec, bool, None]:
```

Comment:
Makes sense that `get_default_selector` returns either `SelectionSpec` or `None`. Is there ever a case where `get_selector` or `get_default_selector` returns `bool`?

Reply:
@jtcohen6 that is correct, though, mypy complained the expected return type for these two should include bool.

Reply:
Ah, classic mypy. Ok!